pkg/kubectl/cmd/annotate.go

```diff
@@ -107,7 +109,7 @@ func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]",
-		Short:   "Update the annotations on a resource",
+		Short:   T("annotate_short"),
```

Comment:
Writing code by having to manually put all your messages in a separate file, defining an id, etc is painful.

What about something like

```
cmd := &cobra.Command{
  (...)
  Short:   T("Update the annotations on a resource"),
```

Then we use `go/ast` to write a smartass Go binary that gets all `CallExpr` of `T(...)` and generate the default (en-US) messages file for us?

```
pkg/kubectl/cmd/annotate#update-the-annotations-on-a-resource: Update the annotations on a resource
pkg/kubectl/cmd/annotate#duplicate-message: A duplicate message
pkg/kubectl/cmd/annotate#duplicate-message2: Another duplicate message
...
```

The message keys can obviously be anything we decide, just has to be uniquely auto-generated given what we have (file name, the actual message string, etc).

And even better, you don't even need the generated file because if a messages file for the wanted language is not present, you just default for what `T` takes as an argument... messages file generation can be used only as a stub for translating the other languages. And while you don't do translations you don't need messages files at all.